### PR TITLE
Fixed variable scope bug.

### PR DESCRIPTION
**Pull request by gautema**

### CODE REVIEW

**Comment by gautema** [comment]
#2 explains bug

File: angular-konami.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ angular.module('konami', [])
 
         // Let user define a custom code.
         if (attr.konamiKeys) {
-          var konamiKeys = scope.$eval(attr.konamiKeys);
```
Comment (lucasconstantino) [inline angular-konami.js:21]: The use of "var" here is not optional; that's how we allow overriding konamiKeys for a directive instance, but keep the default available on the outer scope for future directive instances.

File: angular-konami.js [hunk 2]
```diff
@@ -44,7 +44,7 @@ angular.module('konami', [])
           } else {
             keyIndex = 0;
           }
-        };
+        }
```
Comment (lucasconstantino) [inline angular-konami.js:44]: Oops :)

File: angular-konami.js [hunk 3]
```diff
@@ -60,4 +60,4 @@ angular.module('konami', [])
         scope.$on('$destroy', stopListening);
       }
     };
-  }]);
\ No newline at end of file
+  }]);
```
Comment (lucasconstantino) [inline angular-konami.js:60]: I was about to explain why we should use newlines at the end of files when I understood you were not removing it, but placing. Thanks ;)

**Comment by lucasconstantino** [comment]
So, as explained in the comment on 21, "var" keyword is not optional there. If we simply remove it, we are going to have the default be overridden for future instances once one usage of the directive occurs with custom keys.

The problem must be with the minification method; it is probably giving different names for the `konamiKeys` variables in and out of the the link function, which would indeed cause an undefined variable error. Probably it is not understanding that the second occurrence should actually override the one outside. A definitive solution would be to fallback to more common default settings, having fixed different names in and out.

I'll make a pull-request to you master branch, so that if you accept it, it will modify you own pull-request.

**Comment by gautema** [comment]
I see that my fix didn't work very well. Your new pull request looks good. Thanks for fixing it :)

**Comment by lucasconstantino** [comment]
Thanks to you, for pointing out the bug ;)

**Pull request merged**